.gitmodules

```diff
@@ -9,5 +9,5 @@
 	url = https://chromium.googlesource.com/linux-syscall-support
 [submodule "external/crashpad"]
 	path = external/crashpad
-	url = https://github.com/getsentry/crashpad.git
+	url = https://github.com/Amphaal/crashpad
```

Comment:
Are you sure you want to move crashpad's repo from getsentry to Amphaal's?

Reply:
Nah my bad, this was supposed to be only for master :') I'm preparing an upgrade for crashpad MinGW compatibility (for Linux XCompilation), i'm reverting this asap.